src/sentry/static/sentry/app/utils.tsx

```diff
@@ -61,21 +62,13 @@ function objectMatchesSubset(obj, other, deep) {
   return true;
 }
 
-// XXX(dcramer): the previous mechanism of using _.map here failed
-// miserably if a param was named 'length'
-export function objectToArray(obj) {
-  const result = [];
-  for (const key in obj) {
-    result.push([key, obj[key]]);
-  }
-  return result;
-}
+export const objectToArray = Object.entries;
```

Comment:
Could we refactor this method away? I don't see there being much value in having an alias to stdlib.